Cargo.toml

```diff
@@ -1,3 +1,8 @@
+# See https://github.com/rust-lang/rfcs/blob/master/text/2957-cargo-features2.md
+# Without resolver = 2, using `cargo build --features x` to build `cmd`
+# will _not_ propagate the feature `x` into `cmd`'s direct dependencies.
```

Comment:
But make command already uses `cargo -Zpackage-features`, which should enables features for all packages that have a matched name.

Reply:
@BusyJay `-Zpackage-features` is only used in `scripts/test`. The make command doesn't pass in any `-Z` flags.

anyway both features will soon be stabilized in rust-lang/cargo#8997

Reply:
So can `-Zpackage-features` be removed given that resolver2 is used explicitly?

Reply:
Probably (haven't tried 😂)

Reply:
Can you remove them in this PR?